src/processBadResponse.js

```diff
@@ -13,9 +13,13 @@ function getProcessedErrors(errorMessage, response) {
 
   if (status === 422) {
     try {
-      const { errors } = JSON.parse(errorMessage);
-
-      processedErrors.push(...errors);
+      if (typeof errorMessage === 'object') {
```

Comment:
What is this? This is how BE works in this case? I mean return object instead of json?

Reply:
This function handles different BE scenarios. 
The first one is :
`JSON({ errors: [ { code:" someCode" } ] })`
The second one is: 
`JSON({ errorMessage: JSON({ errors: [ { code: "someCode" ] }))`

As a result we have to parse JSON in some cases twice: so if we recieve an object in this function - that means that we do not have to parse it, but use as it is.